plugins/nextcloud-rule-exclusions-before.conf

```diff
@@ -457,7 +487,7 @@ SecRule REQUEST_FILENAME "@contains /index.php/apps/bookmarks/" \
 # This removes checks on the 'password' and related fields:
 
 # User login password.
-SecRule REQUEST_FILENAME "@contains /index.php/login" \
+SecRule REQUEST_FILENAME "@contains /login" \
```

Comment:
Is `@contains` really needed here? What about `@endsWith`?

Reply:
``@endsWith`` should work fine here as well, I only did ``@contains`` since it was already there.